Automattic/wp-calypso
# Tailored Flows: Redirect and track tailored flows

Related to:
- https://github.com/Automattic/dotcom-forge/issues/9332

## Proposed Changes
* Redirected the paths specified on the linked issue.
* We also track the referrer of the redirecting page so that we can identify them in the future.

## Why are these changes being made?


* We're removing tailored flows and will clean up the code afterward.

## Testing Instructions



* Open the Calypso live link and visit:
  * `/setup/blog`
  * `/setup/link-in-bio`
  * `/setup/videopress`
  * `/setup/sensei`
* They should redirect to `/start/`
* Check t.gif for `calypso_tailored_flows_redirect` events
* After some time, check these events on Tracks.
* Visit:
  * `/setup/free`
* It should redirect to `/start/free/`


https://github.com/user-attachments/assets/86c020df-13c3-4f98-913c-9cd34cd67fb4



## Pre-merge Checklist
